### ISSUE TITLE
Error: "metrics cannot be a standalone element"

### ISSUE THREAD

**Issue by SlavikCA** [traefik/traefik#8430]
### Welcome!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik/issues) and didn't find any.
- [X] Yes, I've searched similar issues on the [Traefik community forum](https://community.containo.us) and didn't find any.

### What did you do?

I followed steps in your official doc to start Traefik, as Docker container as LoadBalancer for Docker nodes.
Everything works.
I connected it to Pilot. It got connected, but there are no any metrics.
So, followed this page:
https://doc.traefik.io/traefik/observability/metrics/overview/

And added this to my YML config:

    metrics: {}

But, after that Traefik fails to start

### What did you see instead?

with `metrics: {}` in the config, Traefik fails to start and I'm getting this error in the log:

> 2021/09/08 23:05:11 command traefik error: metrics cannot be a standalone element (type *types.Metrics)

### What version of Traefik are you using?

2.5.2

### What is your environment & configuration?

```yaml
# Docker configuration backend
providers:
  docker:
    defaultRule: "Host(`{{ trimPrefix `/` .Name }}.docker.localhost`)"
    exposedbydefault: false

entryPoints:
  web:
    address: ":80"
  websecure:
    address: ":443"
    
    
certificatesResolvers:
  myresolver:
    acme:
      email: slavik******
      storage: /etc/traefik/acme.json
      httpChallenge:
        entryPoint: web

# API and dashboard configuration
api:
  insecure: true

log:
  level: INFO

metrics: {}
  
pilot:
    token: "***"
```

I'm not sure, if this is something missing in documentation, or if this is real bug.

**Comment by jbdoumenjou** [comment]
Hello @SlavikCA,

You're right, you have to define at least one of the metrics backend, the doc needs to be fixed.

**Comment by SlavikCA** [comment]
Does it matter which backend?
Or will Pilot service gets same set of metrics regardless of the backend chosen?

**Comment by jbdoumenjou** [comment]
> Or will Pilot service gets same set of metrics regardless of the backend chosen?

Yes,  the Traefik metrics configuration does not impact Pilot metrics.
So, you don't need to configure Traefik metrics to have the Pilot metrics.

**Comment by gsilvapt** [comment]
If that is okay, I can take care of this issue. Any specific wording you guys would like to see? I'm thinking about adding a note after the markup examples saying:

`_Please note at least one metric has to be configured for the configuration to be valid._`.

Please let me know your thoughts.

Thanks!

**Comment by kevinpollet** [comment]
Hello @gsilvapt and thanks for your interest Traefik,

Sure, feel free to work on this issue and to open a pull request targetting the `v2.5` branch. After discussing it with the team, we are thinking that it would be better to remove the `Configuration` section from the `Overview` page.

Does it make sense?

**Comment by gsilvapt** [comment]
Hi @kevinpollet! Absolutely, you guys know better the product than me :) Working on it.

**Comment by rtribotte** [comment]
closed by #8579
